docs/source/usage.rst

```diff
@@ -466,6 +466,38 @@ This can be useful for instance to parse environment variables:
     type: int, value: 3308
 
 
+Extracting lists of keys / values from a dictionary
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some config options that are stored as a ``DictConfig`` may sometimes be easier to manipulate as lists,
+when we care only about the keys or the associated values.
+
+The resolvers ``oc.dict.keys`` and ``oc.dict.values`` simplify such operations by extracting respectively
```

Comment:
Extracting sounds like it's creating a new list.
Maybe something along the lines of "by offering an alternative view of a dictionary's keys or values as a list"

Reply:
> Extracting sounds like it's creating a new list.

It *is* creating a new list.

> Maybe something along the lines of "by offering an alternative view of a dictionary's keys or values as a list"

Done in 70e61fc